assets/js/components/optin.js

```diff
@@ -75,6 +73,8 @@ class Optin extends Component {
 					` );
 				}
 
+				window.googlesitekitTrackingEnabled = !! checked;
```

Comment:
@tofumatt Should this be `window.googlesitekit.admin.trackingOptin` to match the check here? 

https://github.com/google/site-kit-wp/blob/develop/assets/js/util/index.js#L568

`window.googlesitekitTrackingEnabled` is used for the standalone tracking script, I wonder if these should check the same variable (`trackingOptin`)?